vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/Slide.java

```diff
@@ -23,6 +27,9 @@
 import edu.asu.diging.vspace.core.model.ISlide;
 
 @Entity
+@Inheritance(strategy = InheritanceType.JOINED)
```

Comment:
does this work for existing slides?

Reply:
I'm fairly certain you don't use discriminator annotations with joined strategy. you don't need them, there's no need for a column that discriminates between classes. Also still: does this work with existing data?

Reply:
what happens if you remove this? why do you need this? the tables are already setup to use the single table strategy, why this change?